src/features/academic/components/progress/AreaProgressSection.module.scss

```
@@ -1,5 +1,20 @@
 .spacing {
   margin-bottom: spacing.$space-12;
 }
+
+.dualMajorHeader {
+  margin-top: 28px;
+}
+
+.dualMajorTitle {
+  @include typography.body-lg;
+  @include typography.font-weight-bold;
+  margin-bottom: spacing.$space-12;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)